src/winetricks

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+Winetricks/winetricksWinetricks/winetricksWinetricks/winetricksWinetricks/winetricksWinetricks/winetricks#!/bin/sh
 # shellcheck disable=SC2030,SC2031
```

Comment:
Please remove this change.

Reply:
Oops, I don't know why it's here. I was just using the GitHub web interface.... It's fixed now.